met-web/src/routes/AuthenticatedRoutes.tsx

```diff
@@ -88,33 +93,58 @@ const AuthenticatedRoutes = () => {
-                            handle={{
-                                crumb: () => ({
-                                    name: 'Configure',
-                                }),
-                            }}
-                        />
                     </Route>
                     <Route path="old-view" element={<OldEngagementView />} />
-                    <Route path="view" element={<AdminEngagementView />} />
+                    <Route
+                        path="view"
+                        loader={engagementLoader}
+                        handle={{
+                            crumb: async (data: { engagement: Promise<Engagement> }) => {
+                                return data.engagement.then((engagement) => {
+                                    return {
+                                        link: `/engagements/${engagement.id}/details/config`,
+                                        name: engagement.name,
+                                    };
+                                });
+                            },
+                        }}
+                        element={<AdminEngagementView />}
+                    />
+                    <Route index element={<Navigate to="details/config" />} />
+                    <Route path="details">
+                        <Route index element={<Navigate to="config" />} />
+                        {/* Wraps the tabs with the engagement title and TabContext */}
+                        <Route element={<AdminEngagementView />}>
+                            <Route path="config" element={<ConfigTab />} />
+                            <Route path="authoring" element={<AuthoringTab />}></Route>
+                        </Route>
+                        <Route path="authoring" element={<AuthGate allowedRoles={[USER_ROLES.EDIT_ENGAGEMENT]} />}>
+                            <Route element={<AuthoringContext />} action={engagementAuthoringUpdateAction}>
+                                <Route path="banner" element={<AuthoringBanner />} />
+                                {/* <Route path="summary" element={<AuthoringSummary />} />
+									<Route path="details" element={<AuthoringDetails />} />
+									<Route path="feedback" element={<AuthoringFeedback />} />
+									<Route path="results" element={<AuthoringResults />} />
+									<Route path="subscribe" element={<AuthoringSubscribe />} />
+									<Route path="more" element={<AuthoringMore />} /> */}
+                            </Route>
+                        </Route>
+                        <Route path="*" element={<NotFound />} />
+                    </Route>
+                    <Route
+                        path="config/edit"
+                        element={<EngagementConfigurationWizard />}
+                        action={engagementUpdateAction}
+                        handle={{
+                            crumb: () => ({ name: 'Configure' }),
+                        }}
+                    />
```

Comment:
Move the `config/edit` route under `/details` like we discussed
```suggestion
                        </Route>
                        <Route
                            path="config/edit"
                            element={<EngagementConfigurationWizard />}
                            action={engagementUpdateAction}
                            handle={{
                                crumb: () => ({ name: 'Configure' }),
                            }}
                        />
                    </Route>
```